pycbc/workflow/segment.py

```diff
@@ -931,8 +944,11 @@ def get_triggered_coherent_segment(workflow, out_dir, sciencesegs, tag=None):
         else:
             logging.info("This will make OFF-SOURCE symmetrical about trigger "
                          "time.")
-            offsrc = segments.segment(offsrc[0] - offset + excess / 2,
-                                      offsrc[1] - offset - excess / 2)
+            start = int(offsrc[0] - offset + excess / 2)
+            end = int(offsrc[1] - offset - round(float(excess) / 2))
+            offsrc = segments.segment(start, end)
+            #offsrc = segments.segment(offsrc[0] - offset + excess / 2,
+            #                          offsrc[1] - offset - excess / 2)
```

Comment:
Deleted in https://github.com/a-r-williamson/pycbc/commit/cf671262776a9b8410ca88ecc1b445bc25d5d445